librz/core/cmd_analysis.c

```diff
@@ -97,7 +96,6 @@ static const char *help_msg_ab[] = {
 	"ab", " [addr]", "show basic block information at given address",
 	"ab.", "", "same as: ab $$",
 	"aba", " [addr]", "analyze esil accesses in basic block (see aea?)",
-	"abb", " [length]", "analyze N bytes and extract basic blocks",
```

Comment:
Maybe `abb` can be redefined as "analyze N bytes and run `af` as often as possible"?

Reply:
what does that mean?

Reply:
1. Run `af` on the first byte that isn't inside a function.
2. Do step 1 repeatedly until you go above `length`.

`af` might fail but that's a minor detail.

Reply:
I would leave that for another day :/ It is not *that* simple imo.